src/Resources/views/Collector/data_collector.html.twig

```diff
@@ -3,7 +3,7 @@
 {% import _self as helper %}
 
 {% block toolbar %}
-    {% set events = collector.events %}
+    {% set events = collector.getEvents() %}
```

Comment:
Any existing conflict?

Reply:
Not a single one, just a better debug experience IMHO